master/static/srv/get_task.sql

```diff
@@ -1,11 +1,8 @@
-WITH allo AS (
+SELECT tasks.task_id, (SELECT coalesce(jsonb_agg(allo), '[]'::jsonb) FROM (
   SELECT allocation_id, task_id, state, is_ready, start_time, end_time
   FROM allocations
-  WHERE task_id = $1
+  WHERE task_id = tasks.task_id
   ORDER BY end_time DESC NULLS FIRST
```

Comment:
The ordering should be inside the `jsonb_agg` call.

Reply:
👍